async_processor/logger.py

```diff
@@ -1,6 +1,6 @@
 import logging
 
-logger = logging.getLogger("app")
+logger = logging.getLogger("async_processor")
 logger.setLevel(logging.INFO)
 handler = logging.StreamHandler()
 handler.setLevel(logging.INFO)
```

Comment:
Is there some easy way I can set the handler level? maybe pydantic settings env variables?
logger can be kept at DEBUG always